lib/hooks/useFetchAPI.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+export function useFetchAPI(url, option) {
+  const [data, setData] = React.useState([]);
+  const [loading, setLoading] = React.useState(false);
+  const [errorMessage, setErrMsg] = React.useState("");
```

Comment:
Codacy has a fix for the issue: [The variable `errMsg` should be named `errorMessage`. A more descriptive name will do too.](https://app.codacy.com/gh/devntv/useHooks_Apply_projects/pullRequest?prid=9710929)
```suggestion
  const [errorMessage, setErrMsg] = React.useState("");
```